shell/ev-window.c

```diff
@@ -629,8 +627,13 @@ update_chrome_visibility (EvWindow *window)
 	set_widget_visibility (priv->sidebar, sidebar);
 	gtk_revealer_set_reveal_child (GTK_REVEALER (priv->toolbar_revealer), toolbar);
 
-	if (priv->fullscreen_toolbar != NULL) {
-		set_widget_visibility (priv->fullscreen_toolbar, fullscreen_toolbar);
+	if (fullscreen_toolbar)
```

Comment:
Ah, good catch. That was originally much more complicated until I realized I could boil it down to that.